### PR TITLE
[PXS-76131] - Adding fix to send close packet

### PR DESCRIPTION
**Pull request by LunarVagabond**
ensures we can actually transfer files and not just 0 bytes

### CODE REVIEW

File: src/main/java/net/schmizz/sshj/sftp/RemoteFile.java [hunk 1]
```diff
@@ -154,6 +154,9 @@ public void flush() throws IOException {
         @Override
         public void close() throws IOException {
             flush();
+            requester.request(newRequest(PacketType.CLOSE))
+                    .retrieve(requester.getTimeoutMs(), TimeUnit.MILLISECONDS)
+                    .ensureStatusPacketIsOK();
```
Comment (aldiyen) [inline src/main/java/net/schmizz/sshj/sftp/RemoteFile.java]: I think you could just call `super.close()` instead of this.

Comment (LunarVagabond) [reply src/main/java/net/schmizz/sshj/sftp/RemoteFile.java]: I don't think that would work due to this tid bit 
```
The close method of OutputStream does nothing
```

Comment (LunarVagabond) [reply src/main/java/net/schmizz/sshj/sftp/RemoteFile.java]: found a way to clean it up @aldiyen. Good call!

**Comment by aldiyen** [comment]
We determined that this is not actually necessary, and we were just using the SFTP client incorrectly.

**Pull request closed without merging**